src/poliastro/atmosphere/coesa62.py

```diff
@@ -149,24 +150,42 @@ def pressure(self, alt, geometric=True):
 
         # Get base parameters
         i = self._get_index(z, self.zb_levels)
+        zb = self.zb_levels[i]
         hb = self.hb_levels[i]
         Tb = self.Tb_levels[i]
         Lb = self.Lb_levels[i]
         pb = self.pb_levels[i]
 
-        # If Z > 90km, different formulas apply
-        if z <= 90 * u.km:
-            print(z, hb, Tb, Lb, pb)
-            if Lb == 0.0:
-                p = pb * np.exp(-g0 * (h - hb) / Tb / R_air)
-            else:
+        # If Lb is zero then apply eqn 1.2.10-(4)
+        if Lb == 0.0:
```

Comment:
Let us first check if altitude is below 90km and then we can just check if temperature gradient is zero or not. For greater altitudes you can directly apply the polynomial approximations.